tests/lib/wheel.py

```diff
@@ -362,34 +360,46 @@ def make_wheel(
     :param entry_points:
     :param record: if provided and None, then no RECORD file is generated;
         else if a string then sets the content of the RECORD file
+    :param metadata_first: Put the .dist-info metadata at the front of the zip file.
```

Comment:
The wheel spec [explicitly states](https://packaging.python.org/en/latest/specifications/binary-distribution-format/#recommended-archiver-features) that archivers should put the wheel metadata at the *end* of the archive. We absolutely should *not* default to `True` here, and I question why we should make this configurable at all. If it's to test edge cases where pip encounters wheels with the metadata in a non-recommended place, I'd prefer to see the main test suite unchanged, but separate, well-documented tests that explicitly create and test such wheels.

Reply:
Agreed, this should not be configurable.